cfgov/prepaid_agreements/jinja2/prepaid_agreements/detail.html

```diff
@@ -0,0 +1,134 @@
+{%- endblock %}
+
+{% block content_main %}
+
+{% set default_text = 'No information provided' %}
+<h1>{{ product.name }}</h1>
+<div class="content-l">
+        <div class="content-l_col content-l_col-1-2">
+            <dl>
+                <dt><h2>Issuer name</h2></dt>
+                <dd>{{ product.issuer_name }}</dd>
+                <dt><h2>Prepaid product type</h2></dt>
+                <dd>{{ product.prepaid_type if product.prepaid_type else default_text }}</dd>
+            </dl>
+        </div>
+        <div class="content-l_col content-l_col-1-2">
+            <dl>
+                <dt><h2>Program manager</h2></dt>
+                <dd>{{ product.program_manager if product.program_manager else default_text }}</dd>
+                <dt><h2>Current status</h2></dt>
+                <dd>{{ product.status if product.status else default_text }}</dd>
+            </dl>
+        </div>
+    </div>
+        <dl>
+            <dt><h2>Other relevant parties</h2></dt>
+            <dd>
+            {% if product.other_relevant_parties %}
+                {% if product.other_relevant_parties | length > 500 %}
+                    <div data-qa-hook="expandable"
+                         class="o-expandable">
+                        <div class="o-expandable_content">
+                {% endif%}
+                {{ product.other_relevant_parties | replace('\n', '; ') }}
+                {% if product.other_relevant_parties | length > 500 %}
+                        </div>
+                        <button class="a-btn a-btn__link
+                                        o-expandable_header
+                                        o-expandable_target">
+                            <span class="o-expandable_link">
+                                <span class="o-expandable_cue o-expandable_cue-open">
+                                    Show full list of relevant parties
+                                    {{ svg_icon('plus-round') }}
+                                </span>
+                                <span class="o-expandable_cue o-expandable_cue-close">
+                                    Hide full list of relevant parties
+                                    {{ svg_icon('minus-round') }}
+                                </span>
+                            </span>
+                        </button>
+                    </div>
+                {% endif %}
+            {% else %}
+                {{ default_text }}
+            {% endif %}    
+            </dd>         
+        </dl>
+
+{% set agreements = product.prepaidagreement_set.all() %}
+{% if agreements | length %}
+
+    <div class="block u-mt45">
+        <h2 class="h3"> Download most recent agreement files</h2>
+        {% set agreement = agreements | first  %}
+        <div class="block block__sub block__flush-top">
+            <p class="u-mb5"><span>Agreement effective date:</span> {{ agreement.effective_date }}</p>
+            <p class="break-word">
```

Comment:
Should this be `u-break-word`?

Reply:
Thanks for catching that -- updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96